example/basic/main.go

```diff
@@ -1,156 +1,56 @@
+// This is an example program that makes an HTTP request and prints response
+// headers. Whenever a request fails the error is reported to Sentry.
+//
+// Try it by running:
+//
+// 	go run main.go
+// 	go run main.go https://sentry.io
+// 	go run main.go bad-url
+//
+// To actually report events to Sentry, set the DSN either by editing the
+// appropriate line below, or set the environment variable
+// SENTRY_DSN=<your-dsn>. Replace "<your-dsn>" with a proper DSN you can find in
```

Comment:
Same `in` and `or` changes in both comments apply here

Reply:
Reworked the copy.